build.sbt

```diff
@@ -3,17 +3,15 @@ import Dependencies._
 ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.4.4"
 
 libraryDependencies ++= Seq(
-  Library.Play.ws,
-  Library.Play.test,
-  Library.Play.json,
-  Library.Play.jsonJoda,
-  Library.Specs2.matcherExtra,
-  Library.Specs2.mock,
-  Library.Specs2.core,
-  Library.Utils.jodaTime,
-  Library.Play.playAkkaHttpServer,
-  Library.DataswiftModels.hat,
-  Library.DataswiftModels.hatPlay,
+  Library.playWs,
+  Library.playJson,
+  Library.playJsonJoda,
+  Library.jodaTime,
+  Library.hatPlayModels,
+  Library.Specs2.matcherExtra % Test,
+  Library.Specs2.mock % Test,
+  Library.Specs2.core % Test,
+  Library.playTest,
```

Comment:
should this not have `Test` scope too ?

Reply:
its current set to compile scope with a comment

```"compile" // Used by the MockHatServer ```

this was existing code i will investigate

Reply:
Mock Hat server is NOT part of the test package and relies on PlayTest
This is existing code, what the purpose of MockHatServer is and what uses it should potentially be investigated at some stage but i think it is outside of the scope of this PR.
Ive moved all scoping to the build.sbt so this is more obvious

Reply:
*bump*